rules/linux/credential_access_potential_successful_linux_ftp_bruteforce.toml

```diff
@@ -22,19 +22,34 @@ language = "eql"
 license = "Elastic License v2"
 name = "Potential Successful Linux FTP Brute Force Attack Detected"
 note = """## Setup
-This rule requires the use of the `auditd_manager` integration. `Auditd_manager` is a tool designed to simplify and enhance the management of the audit subsystem in Linux systems. It provides a user-friendly interface and automation capabilities for configuring and monitoring system auditing through the auditd daemon. With `auditd_manager`, administrators can easily define audit rules, track system events, and generate comprehensive audit reports, improving overall security and compliance in the system. The following steps should be executed in order to install and deploy `auditd_manager` on a Linux system. 
-```
-Kibana -->
-Management -->
-Integrations -->
-Auditd Manager -->
-Add Auditd Manager
-```
-`Auditd_manager` subscribes to the kernel and receives events as they occur without any additional configuration. However, if more advanced configuration is required to detect specific behavior, audit rules can be added to the integration in either the "audit rules" configuration box or the "auditd rule files" box by specifying a file to read the audit rules from. 
-```
-For this detection rule no additional audit rules are required to be added to the integration. 
-```
-Add the newly installed `auditd manager` to an agent policy, and deploy the agent on a Linux system from which auditd log files are desirable.
+
```

Comment:
Suggestion: add " This rule requires either `auditd_manager` or Auditbeat integration.